recipes/pynose/LICENSE

```diff
@@ -0,0 +1,504 @@
+		  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Both licenses are in the source distribution so you don't need to include a copy here in the recipe.